ArduPlane/system.cpp

```diff
@@ -149,7 +149,8 @@ void Plane::init_ardupilot()
 
     // setup frsky
 #if FRSKY_TELEM_ENABLED == ENABLED
-    frsky_telemetry.init(serial_manager);
+    // setup frsky, and pass a number of parameters to the library
+    frsky_telemetry.init(serial_manager, (uint8_t *)&control_mode);
```

Comment:
Thanks for taking the time to respond. That's how we had it before but check what Randy said here (point 3): https://github.com/ArduPilot/ardupilot/pull/3434#issuecomment-199559209